src/components/App/App.js

```diff
@@ -55,7 +69,7 @@ function App() {
     signup(email, username, password)
       .then((res) => {
         res.json().then((res) => {
```

Comment:
* `res.json()` should not be duplicated in your requests in `App`. Please, use it only in `_customFetch` or `_checkResponse` otherwise you'll get bugs or duplication of the logics